### PR TITLE
Fix gem build licence warning

### PR DESCRIPTION
**Pull request by CrCs2O4**
When building gem created with Methadone there is a warning:

``` bash

WARNING: license value 'mit' is invalid.  Use a license identifier from
http://spdx.org/licenses or 'Nonstandard' for a nonstandard license.
```

Thanks for awesome app!

### CODE REVIEW

File: bin/methadone [hunk 1]
```diff
@@ -69,7 +69,7 @@ main do |app_name|
   if gemspec_content =~ /(^\s*#{gem_variable}\.license\s*=\s*).*/
     gemspec_content.gsub!(/(^\s*#{gem_variable}\.license\s*=\s*).*/,"\\1#{license.to_s.inspect}")
   else
-    gemspec_content.gsub!(/(^\s*#{gem_variable}\.name\s*=\s*.*)/,"\\1\n  #{gem_variable}.license=#{license.to_s.inspect}")
+    gemspec_content.gsub!(/(^\s*#{gem_variable}\.name\s*=\s*.*)/,"\\1\n#{"  #{gem_variable}.license".ljust(20)} = #{license.to_s.inspect.upcase}")
```
Comment (davetron5000) [inline bin/methadone]: Is this needed?

**Comment by davetron5000** [comment]
d'oh.  thanks for finding this!

**Comment by CrCs2O4** [comment]
@davetron5000 you need something more from me to merge this PR?

File: bin/methadone (hunk 1 above)
Comment (davetron5000) [inline bin/methadone]: @CrCs2O4 what is the `ljust` for here?

Comment (CrCs2O4) [inline bin/methadone]: `ljust` align license string with others

Before PR:
ljust

After PR:
ljust

Comment (davetron5000) [inline bin/methadone]: ok, cool.  make sense.

**Comment by davetron5000** [comment]
Released as https://github.com/davetron5000/methadone/releases/tag/v1.9.3

Thanks again!

**Pull request merged**